dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/canvas.scss

```diff
@@ -34,14 +34,35 @@
 
     .minimap {
       position: absolute;
-      width: 300px;
-      height: 200px;
-      right: 10px;
-      bottom: 10px;
+      right: 0px;
+      bottom: 0px;
       border: dashed 1px #e4e4e4;
       z-index: 9;
     }
 
+    .scale-slider{
+      position: absolute;
+      height: 140px;
+      width: 70px;
+      right: 0px;
+      bottom: 140px;
+      z-index: 9;
+      display: flex;
+      justify-content: center;
+
+      /deep/ .el-slider__runway{
```

Comment:
It is recommended to replace `/deep/` with `::v-deep`.

Reply:
done.